src/main/java/com/hellosign/sdk/resource/TemplateSignatureRequest.java

```diff
@@ -195,7 +195,8 @@ public void addCustomField(CustomField field) {
     /**
      * Adds the value to fill in for a custom field with the given field name.
      * 
-     * @param fieldName String field name to be filled in
+     * @param fieldName String name (or "Field Label") of the custom field
+     *        to be filled in. The "api_id" can also be used instead of the name.
      * @param value String value
      */
     public void setCustomFieldValue(String fieldName, String value) {
```

Comment:
For readability, maybe changing the name of `fieldName` to `fieldNameOrApiId`. I come from a land of no javadoc and letting code be self descriptive :)